packages/slice-machine/src/components/Window/Window.css.ts

```diff
@@ -0,0 +1,156 @@
+import { colors, sprinkles, vars } from "@prismicio/editor-ui";
```

Comment:
ℹ️ I will not really get into the specific of this file 😅 If you feel I should for the review let me know.